### PR TITLE
[Merged by Bors] - fix: fix toolchain specification

### PR DESCRIPTION
**Pull request by xxchan**
Previous filename and channel name are wrong

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.68"
```
Comment (sehz) [inline rust-toolchain.toml:2]: thanks for changes.  this should be `1.68.2`

**Review by sehz — CHANGES_REQUESTED** [review]

File: rust-toolchain.toml (hunk 1 above)
Comment (xxchan) [inline rust-toolchain.toml:2]:
```suggestion
channel = "1.68.2"
```

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**